openapi-cli/src/test/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmdTest.java

```diff
@@ -36,7 +36,8 @@
     private static final Path RES_DIR = OpenAPICommandTest.getResourceFolderPath();
     Path resourcePath = Paths.get(System.getProperty(USER_DIR));
 
-    @Test(description = "Test openapi gen-service for successful service generation with inline request body type")
+    @Test(description = "Test openapi gen-service for successful service generation with inline request body type",
+            enabled = false)
```

Comment:
Are we tracking the disabled test cases ? If not lets create some issues to track them

Reply:
Created the issue for tracking disable tests 
[Issue](https://github.com/ballerina-platform/ballerina-openapi/issues/40l)